photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala

```diff
@@ -82,10 +86,10 @@ object Params {
     val defaultParams = Params()
     val parser = new OptionParser[Params]("GLMix-Scoring-Params") {
       opt[String]("input-data-dirs")
-        .required()
-        .text("Input directories of data to be scored. Multiple input directories are also accepted if they are " +
-          "separated by commas, e.g., inputDir1,inputDir2,inputDir3.")
-        .action((x, c) => c.copy(inputDirs = x.split(",")))
+          .required()
+          .text("Input directories of data to be scored. Multiple input directories are also accepted if they are " +
+              "separated by commas, e.g., inputDir1,inputDir2,inputDir3.")
+          .action((x, c) => c.copy(inputDirs = x.split(",")))
```

Comment:
@XianXing I can't find anything about this in the style guide, but I believe the previous indentation style (2-space) is the generally accepted way when you have a multi-line method chain like this.